<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout_3"/>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListView" name="listView"/>
       </item>
       <item>
        <widget class="QTreeView" name="treeView"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Optio&amp;ns</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionScan"/>
    <addaction name="actionPlayList"/>
    <addaction name="actionPlay"/>
    <addaction name="actionPause"/>
    <addaction name="actionSkip_Foward"/>
    <addaction name="actionSkip_Backward"/>
    <addaction name="actionStop"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSkip_Backward"/>
   <addaction name="actionPlay"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="actionSkip_Foward"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/document-open.png</normaloff>:/document-open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/application-exit.png</normaloff>:/application-exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/media-playback-start.png</normaloff>:/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>P&amp;lay</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/media-playback-pause.png</normaloff>:/media-playback-pause.png</iconset>
   </property>
   <property name="text">
    <string>Pa&amp;use</string>
   </property>
  </action>
  <action name="actionSkip_Foward">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/media-skip-forward.png</normaloff>:/media-skip-forward.png</iconset>
   </property>
   <property name="text">
    <string>Skip &amp;Foward</string>
   </property>
  </action>
  <action name="actionSkip_Backward">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/media-skip-backward.png</normaloff>:/media-skip-backward.png</iconset>
   </property>
   <property name="text">
    <string>Skip &amp;Backward</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/media-playback-stop.png</normaloff>:/media-playback-stop.png</iconset>
   </property>
   <property name="text">
    <string>St&amp;op</string>
   </property>
  </action>
  <action name="actionScan">
   <property name="text">
    <string>&amp;Scan</string>
   </property>
  </action>
  <action name="actionPlayList">
   <property name="text">
    <string>&amp;PlayList</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
